pages/_document.js

```diff
@@ -41,13 +41,21 @@ export default class GDLDocument extends Document {
 
     return (
       <html lang={this.props.language}>
-        <Head>
-          <title>Global Digital Library</title>
-          <meta name="viewport" content="width=device-width, initial-scale=1" />
-          <meta httpEquiv="x-ua-compatible" content="ie=edge" />
-          <script dangerouslySetInnerHTML={{ __html: `window.${config.GLOBAL_VAR_NAME} = '${process.env.GDL_ENVIRONMENT || 'test'}';` }} />
-          {this.props.styleTags}
-        </Head>
+      <Head>
+        <title>Global Digital Library</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta httpEquiv="x-ua-compatible" content="ie=edge" />
+        <script dangerouslySetInnerHTML={{ __html: `window.${config.GLOBAL_VAR_NAME} = '${process.env.GDL_ENVIRONMENT || 'test'}';` }} />
+        <script dangerouslySetInnerHTML={{__html: `
```

Comment:
We should use the *async* method to load the script, see https://developers.google.com/analytics/devguides/collection/analyticsjs/

We don't support IE9 anyways, and we can use this method in browsers such as Samsung Mobile and UC, https://caniuse.com/#feat=script-async

Reply:
Fixed